tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "compilerOptions": {
-    "moduleResolution": "node",
-    "module": "commonjs",
-    "target": "es2019",
-    "lib": ["es2020"],
+    "moduleResolution": "Node16",
+    "module": "Node16",
```

Comment:
I'll have to check whether sf cli can consume this

Reply:
I think it should, the reason I changed it in this specific way is because of the dev-config here: https://github.com/forcedotcom/dev-config/blob/main/tsconfig-esm.json

The template that Salesforce provides for CLI plugins extends this config by default.